app/utils/helpers.tsx

```diff
@@ -288,3 +288,12 @@ export function getFormStageInfo({ isAdmin, formStage, teams }: Args) {
     schemas,
   };
 }
+
+export function canDeleteMember(members: User[], memberId?: string) {
+  if (members.length === 1) return false;
+  const memberToDelete = members.find((member) => member.id === memberId);
+  const memberIsLastAdmin =
+    members.filter((member) => member.role === 'admin').length === 1 && memberToDelete?.role === 'admin';
```

Comment:
do all `member`s have 'roles'?  If we're trying to delete a role-less member this could error out.  Just confirming since there is a check at the end of the line `memberToDelete?.role`.

Reply:
good question, yes all members are required to have roles so the members array is safe. I put the '?' check on the member to delete because the `members.find((member) => member.id === memberId);` line could return null if noone has that ID

Reply:
Oh good question, yes the role is required on the members array. For the `memberToDelete` because that came from `members.find((member) => member.id === memberId);` it could be null if no matching id was found